src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2854,5 +2854,9 @@ Include=Include
 Exclude=Exclude
 Include\ or\ exclude\ cross-referenced\ entries=Include or exclude cross-referenced entries
 Would\ you\ like\ to\ include\ cross-reference\ entries\ in\ the\ current\ operation?=Would you like to include cross-reference entries in the current operation?
+
 Entries\ copied\ successfully,\ including\ cross-references.=Entries copied successfully, including cross-references.
 Entries\ copied\ successfully,\ without\ cross-references.=Entries copied successfully, without cross-references.
+
+contains\ a\ URL=contains a URL
```

Comment:
Should be moved to some other integrity check outputs, but I leave this as future work to keep things going.